conf/app.routes

```diff
@@ -49,6 +49,7 @@ POST         /property-links
 
 GET          /owner/agents                                                                    @uk.gov.hmrc.voapropertylinking.controllers.PropertyLinkingController.getMyOrganisationsAgents()
 GET          /owner/property-links                                                            @uk.gov.hmrc.voapropertylinking.controllers.PropertyLinkingController.getMyOrganisationsPropertyLinks(searchParams: GetMyOrganisationPropertyLinksParameters, params: Option[models.PaginationParams], organisationId: Option[Long] ?= None)
+GET          /owner/property-links-count                                                      @uk.gov.hmrc.voapropertylinking.controllers.PropertyLinkingController.getMyOrganisationsPropertyLinksCount()
```

Comment:
the url should be showing the part of the resource we are going for e.g owner/property-links/count.

Reply:
I will change this